irc/weechat/Portfile

```diff
@@ -63,6 +63,10 @@ license_noconflict  asciidoctor
 
 patchfiles          no-extra-gcc-warnings.patch
 
+# core-crypto.c:533: error: implicit declaration of function ‘htobe64’
+# https://github.com/weechat/weechat/pull/2216
+patchfiles-append   patch-fix-htobe64.diff
```

Comment:
Should this just be added to the `patchfiles` above rather than using `patchfiles-append`?

Reply:
@cardi The idea was simply that it is easier to drop a separate chunk (for someone looking into the PR or into the history) once the fix is hopefully merged to upstream. Functionally these are identical.